scvi/model/_scanvi.py

```diff
@@ -136,6 +136,25 @@ def __init__(
         )
         self.init_params_ = self._get_init_params(locals())
 
+    @classmethod
+    def load_pretrained_scvi(
+        cls,
+        scvi_model,
+        unlabeled_category,
+        adata=None,
+        use_gpu=None,
+        **scanvi_kwargs,
+    ):
```

Comment:
needs a docstring and typing

Reply:
also should check if scvi model has been trained, and provide a logger warning if it hasn't.